app/index.html

```diff
@@ -19,6 +19,7 @@
 
     <script src="assets/vendor.js"></script>
     <script src="assets/super-rentals.js"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?v=3.23"></script>
```

Comment:
For simplicity, I'm globally importing the maps script.  Open to better ways with maps.